README.md

```diff
@@ -247,15 +247,15 @@ module.exports = function() {
 
 ```js
 // app/initializers/ember-cli-simple-store.js
-import SimpleStore from 'ember-cli-simple-store/store';
+import SimpleStore from '../services/simple-store';
 
 export default {
     name: 'override-simple-store',
     initialize() {
         var app = arguments[1] || arguments[0];
-        app.register('simpleStore:main', SimpleStore);
-        app.inject('controller', 'simpleStore', 'simpleStore:main');
-        app.inject('route', 'simpleStore', 'simpleStore:main');
+        app.register('service:simple-store', SimpleStore);
```

Comment:
We should probably make this the default implementation of the initializer and then we can remove the documentation all together along with the disableInjection functionality.  Since this is 4.0, we're free to make this change.  So long as no one was accessing store:main directly, it would be a safe change to make.